tests/runfiles/bclone.run

```diff
@@ -8,9 +8,7 @@
 # source.  A copy of the CDDL is also available via the Internet at
 # http://www.illumos.org/license/CDDL.
 #
-# This run file contains all of the common functional tests.  When
-# adding a new test consider also adding it to the sanity.run file
-# if the new test runs to completion in only a few seconds.
```

Comment:
Did you want this change included in this PR?

Reply:
Not really part of this PR, but I was hoping to smuggle it in instead of creating a separate PR for it:)